docs/source/what-is-mlflow.rst

```diff
@@ -0,0 +1,51 @@
+.. _what-is-mlflow:
+
+What is MLflow?
+===============
+
+MLflow is a versatile, expandable, open-source platform for managing machine learning workflows and artifacts across the machine learning lifecycle. It has predefined flavors for many popular ML libraries, but can be used with any library, algorithm, or deployment tool. It is designed to be extensible, so you can write plugins to support new workflows, libraries, and tools.
+
+.. figure:: _static/images/what-is-mlflow/mlflow-overview.png
+    :width: 100%
+
+MLflow is composed of five components:
+
+- `MLflow Tracking <https://www.mlflow.org/docs/latest/tracking.html>`_: An API and UI for logging parameters, code versions, metrics, and output files when running your machine learning code and for later visualizing the results.
+
+- `MLflow Projects <projects>`_: A standard format for packaging reusable data science code that can be run with different parameters to train models, visualize data, or perform any other data science task.
+
+- `MLflow Models <models>`_: A model packaging format and tools that let you easily deploy the same model (from any ML library) to batch or real-time scoring on platforms such as Docker, Apache Spark, Azure ML and AWS SageMaker.
```

Comment:
```suggestion
- `MLflow Models <models>`_: A model packaging format and suite of tools that let you easily deploy a trained  model (from any ML library) for batch or real-time inference on platforms such as Docker, Apache Spark, Azure ML and AWS SageMaker.
```

Reply:
I like @BenWilson2 's suggestion, but can we also mention Databricks as a deployment target?